### PR TITLE
[bitnami/wordpress] Add support for SANs on certs

### LINKED ISSUE #4924: [bitnami/wordpress] -- extraTls block creates additional cert rather than bundle into main
**Which chart**:
bitnami/wordpress (latest version)

**Describe the bug**
I want to create a cert that has the main domain (for example www.joebloggs.com) and some subdomains (for example, blog.joebloggs.com and archive.joebloggs.com)

When using the extraTls block, it does not create an ingress that I would expect for a multi-name SSL cert. 

**To Reproduce**

Create an installation with the following ingress override (values.yaml) (annotations optional, replace with a domain you control):

```
ingress:
  enabled: true
  certManager: true
  tls: true
  hostname: www.joebloggs.com
  annotations:
    kubernetes.io/ingress.class: "nginx"
    cert-manager.io/cluster-issuer: "letsencrypt-production"
    nginx.ingress.kubernetes.io/proxy-body-size: 1G
```

This will create a cert successfully. With the ingress formatted like this:

```
    tls:
    - hosts:
      - www.joebloggs.com
      secretName: www.joebloggs.com-tls
```

This is fine and behaves as expected

Now I want to add two additional names to the cert. According to the readme.md (https://github.com/bitnami/charts/tree/master/bitnami/wordpress) and the example in the values.yaml (https://github.com/bitnami/charts/blob/b7836ee7dcb1067e466b772ae626fbc0b4c14cd2/bitnami/wordpress/values.yaml#L481) I use this updated values.yaml override:

```
ingress:
  enabled: true
  certManager: true
  tls: true
  extraTls:
    - hosts:
        - blog.joebloggs.com
        - archive.joebloggs.com
      secretName: extra-ssl-tls      
  hostname: www.joebloggs.com
  annotations:
    kubernetes.io/ingress.class: "nginx"
    cert-manager.io/cluster-issuer: "letsencrypt-production"
    nginx.ingress.kubernetes.io/proxy-body-size: 1G
```

This creates the following ingress block:

```
    tls:
    - hosts:
      - www.joeblogss.com
      secretName: www.joebloggs.com-tls
    - hosts:
      - blog.joebloggs.com
      - archive.joebloggs.com
      secretName: extra-ssl-tls 
```

This would create two TWO certs -- one for www.joebloggs.com and one for blog.joebloggs.com and archive.joebloggs.com.

Is there a reason why we can't do this instead:

```
    tls:
    - hosts:
      - www.joeblogss.com
      - blog.joebloggs.com
      - archive.joebloggs.com
      secretName: www.joebloggs.com-tls
```

This would create one cert rather than two, with all the names covered?

**Expected behavior**
One cert create covering the main host and the alternative names

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.3.4", GitCommit:"a61ce5633af99708171414353ed49547cf05013d", GitTreeState:"clean", GoVersion:"go1.14.9"}

```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.6", GitCommit:"dff82dc0de47299ab66c83c626e08b245ab19037", GitTreeState:"clean", BuildDate:"2020-07-15T16:58:53Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18+", GitVersion:"v1.18.9-eks-d1db3c", GitCommit:"d1db3c46e55f95d6a7d3e5578689371318f95ff9", GitTreeState:"clean", BuildDate:"2020-10-20T22:18:07Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}

```

**Additional context**

### PR DESCRIPTION
**Pull request by huang-jy**
**Description of the change**

Adds support to add additional hosts to the "main" cert requested by the chart as opposed to creating a separate cert. This PR implements a new field: `ingress.additionalHostNames` and can be used as follows:

Example: No additional names (current behaviour)
```
ingress:
  enabled: true
  certManager: true
  tls: true
  hostname: www.joebloggs.net
```

Example: One additional host name
```
ingress:
  enabled: true
  certManager: true
  tls: true
  hostname: www.joebloggs.net
  alternativeHostNames:
    - blog.joebloggs.net
```

Example: Multiple additional host names
```
ingress:
  enabled: true
  certManager: true
  tls: true
  hostname: www.joebloggs.net
  alternativeHostNames:
    - blog.joebloggs.net
    - archive.joebloggs.net
```

**Benefits**

Minimises the number of certs generated by the chart.

**Possible drawbacks**

The range claused used in the change prevents to usage of field evaluation, so the change does not respect the `{{- if .Values.ingress.extraPaths }}` block in the additional rules it creates

**Applicable issues**

  - fixes #4924 

**Additional information**

N/A

**Checklist** 
- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/wordpress/templates/ingress.yaml [hunk 1]
```diff
@@ -47,6 +59,9 @@ spec:
     {{- if .Values.ingress.tls }}
     - hosts:
         - {{ .Values.ingress.hostname }}
+        {{- range .Values.ingress.alternativeHostNames }}
```
Comment (dani8art) [inline bitnami/wordpress/templates/ingress.yaml:62]: Why don't we use `extraHosts` directly? we would avoid introducing a new value.

**Review by dani8art — CHANGES_REQUESTED** [review]
Hi @huang-jy thanks for the PR, please, take a look at my comment

File: bitnami/wordpress/templates/ingress.yaml (hunk 1 above)
Comment (huang-jy) [reply bitnami/wordpress/templates/ingress.yaml:62]: Good spot. Let me try that

**Comment by huang-jy** [comment]
Pushed an update and tested.

File: bitnami/wordpress/values.yaml [hunk 2]
```diff
@@ -446,6 +446,13 @@ ingress:
   ##
   path: /
 
+  ## Alternative host names. This will add SANs (Subject Alternative Names) to the cert so
+  ## that one SSL certificate covers multiple names. The hosts here will map to ingress.path.
+  ##
+  ## If you want a separate certificate, use ingress.extraTls.hosts, ingress.extraTls.secretName
+  ## for the cert, and ingress.extraHosts.name and ingress.extraHosts.path for the ingress mapping
+  alternativeHostNames: []
```
Comment (dani8art) [inline bitnami/wordpress/values.yaml:454]: Please, remove this

**Review by dani8art — APPROVED** [review]
LGTM, just a single request

File: bitnami/wordpress/values.yaml (hunk 2 above)
Comment (huang-jy) [reply bitnami/wordpress/values.yaml:454]: Ah yes, I missed that sorry. Will remove

Comment (huang-jy) [reply bitnami/wordpress/values.yaml]: This is now removed from latest commit

**Review by dani8art — APPROVED** [review]
LGTM!

**Pull request merged**